examples/vue/src/styles.scss

```diff
@@ -37,7 +37,7 @@ h1.amplify-heading {
 }
 
 h3.amplify-heading {
-  font-weight: 600 !important;
```

Comment:
The Vue text was bolder than the React text before.

Reply:
👍